examples/models/file/daylight.py

```diff
@@ -1,3 +1,13 @@
+''' A plot shading the daylight hours in a city's local time (which are
```

Comment:
"shading" seems to me an uncommon wording in this context. Maybe use "showing"?

Reply:
That works for me.